swedish-oidc-profile.md

```diff
@@ -472,18 +502,22 @@ An OpenID Provider compliant with this profile MUST NOT release any identity cla
 
 Section 5.4 of \[[OpenID.Core](#openid-core)\] states:
 
-> The Claims requested by the `profile`, `email`, `address`, and `phone` scope values are returned from the UserInfo Endpoint, as described in Section 5.3.2, when a `response_type` value is used that results in an Access Token being issued. However, when no Access Token is issued (which is the case for the `response_type` value `id_token`), the resulting Claims are returned in the ID Token.
+> The Claims requested by the `profile`, `email`, `address`, and `phone` scope values are returned from the UserInfo Endpoint, as described in Section 5.3.2, when a `response_type` value is used that results in an Access Token being issued. However, when no Access Token is issued (which is the case for the `response_type` value `id_token`), the resulting Claims are returned in the ID Token. 
 
 \[[OpenID.Core](#openid-core)\] basically assumes that the user identity is delivered in the `sub` claim that is part of the ID token, and all other attributes are complementary attributes that may be fetched in a later call to the UserInfo endpoint. 
 
 The Swedish OpenID Connect profile takes another approach regarding the primary user identity, and the primary user identity is most often represented by a claim delivered as part of a requested scope. Therefore, this profile, requires that if any of the scopes defined in section 3 of \[[AttrSpec](#attr-spec)\] are requested the corresponding claims MUST be delivered in the ID token<sup>3</sup>.
 
+"Authentication Information Claims" as defined in section 2.3 of \[[AttrSpec](#attr-spec)\] are claims
+describing an authentication event, and are not user identity claims. Therefore, any of these claims
+MUST be delivered in the ID token and not from the UserInfo endpoint.
+
```

Comment:
Why do we have any opinion whether there is a UserInfo endpoint and what it provides?
If we do, should we use proper requirement language? (MUST NOT)